lib/bolt/outputter/human.rb

```diff
@@ -25,24 +25,24 @@ def remove_trail(string)
         string.sub(/\s\z/, '')
       end
 
-      def print_event(node, event)
+      def print_event(target, event)
         case event[:type]
         when :node_start
-          print_start(node)
+          print_start(target)
         when :node_result
-          print_result(node, event[:result])
+          print_result(event[:result])
```

Comment:
Since result has a target, target is redundant. This creates a kind of asymmetry with start events that I don't much like.

Reply:
Should we just make `:target` a part of the node_start event?